dataprofiler/profilers/profiler_options.py

```diff
@@ -323,19 +332,32 @@ def _validate_helper(self, variable_path='NumericalOptions'):
 
         errors = super()._validate_helper(variable_path=variable_path)
         for item in ["histogram_and_quantiles", "min", "max", "sum",
-                     "variance"]:
+                     "variance", "skewness", "kurtosis"]:
             if not isinstance(self.properties[item], BooleanOption):
                 errors.append("{}.{} must be a BooleanOption."
                               .format(variable_path, item))
             else:
                 errors += self.properties[item]._validate_helper(
                     variable_path=variable_path + '.' + item)
 
-        if not self.properties["sum"].is_enabled and \
-                self.properties["variance"].is_enabled:
+        # Error checks for dependent calculations
+        sum_disabled = not self.properties["sum"].is_enabled
+        var_disabled = not self.properties["variance"].is_enabled
+        skew_disabled = not self.properties["skewness"].is_enabled
+        kurt_disabled = not self.properties["kurtosis"].is_enabled
+        if sum_disabled and not var_disabled:
             errors.append("{}: The numeric stats must toggle on the sum "
                           "if the variance is toggled on."
                           .format(variable_path))
+        if (sum_disabled or var_disabled) and not skew_disabled:
+            errors.append("{}: The numeric stats must toggle on the "
+                          "sum and variance if skewness is toggled on."
+                          .format(variable_path))
+        if (sum_disabled or var_disabled or skew_disabled) \
+            and not kurt_disabled:
```

Comment:
can we still calculate kurtosis without the need of skewness?

Reply:
The [algorithm](https://en.wikipedia.org/wiki/Algorithms_for_calculating_variance#Higher-order_statistics) for computing kurtosis relies on knowing M3 (third central moment without dividing by N), which I found is extracted from skewness. Not sure how else it could be done, thoughts?

Reply:
What I meant is, then we can find kurtosis even we have `skew_disabled` is True, as we can find skewness and update the kurtosis like you mentioned

Reply:
I guess this is possible, I just added these error checks to keep consistent with the original error check for variance. Although you can still technically compute variance without sum enabled (just need to compute sum in the background but not report it), it still raises an error so I figured the same should be for skewness/kurtosis for consistency. What do you think?

Reply:
If any of the features require something else, it should be dependent on it being enabled. We shouldn't calculate it externally.

Reply:
@AnhTruong, hence if the dependencies are correct, it should be done as is.

Reply:
I still see this is a potential use case, but as we're having the consistency across other stats, we can raise a warning to reset other stats if the dependent stat is needed. But for now, we haven't had a user's request for such a case, then we can leave it as is.